src/utils/SpokeUtils.ts

```diff
@@ -21,7 +21,7 @@ type BlockTag = providers.BlockTag;
  */
 export function populateV3Relay(
```

Comment:
Is there a way we currently test this function or is it used only in prod?

Reply:
Seems like it's only used in prod - we use it in the relayFeeCalculator.